### PR TITLE
Scipaper rebalancing: Nitrium and halon shell removal. Nitrous added. Emphasis on BZ.

### PR DESCRIPTION
**Pull request by vincentiusvin**
## About The Pull Request
Similar in spirit to #65707, with some more changes.

Restructured the gaseous experiments to:
1. Nitrous (practice experiment)
2. BZ (mainstay experiment)
3. Hyper-Nob (lategame/once-in-a-while experiment)

Added a mining partner.

Moved adv weaponry lock to normal weaponry under reactionless. Toned down t3 reactionless.

BZ locks adv engi. Medbay unbridled by toxin gasses now.

Removed Xenobio's BZ Can.

## Why It's Good For The Game
My original intent with papers was expanding the difficulty range of toxins. Both to things harder than tritium (nob, nitrium, etc) and also to things easier than tritium (bz, reactionless, etc). 

In that process, I feel that i strayed a bit to the harder side, this PR is an attempt to tone down the overall difficulty of some of the gaseous experiments a notch. 

Nitrous now takes place of the old BZ, BZ takes place of old nitrium/halon, and noblium stays because it's difficulty is in a pretty good spot for a relatively unimportant but nice to have tech.

While we're at it, I also added more emphasis to BZ production to toxins instead of tritium. This will hopefully incentivize people to try the department out. There is a risk of this being a bit of a chore, but I believe that the relevant atmos gameplay loop is strong enough to have it be fun. You need to check on the chamber, turn on pipes, adjust the input rate, and many more that makes it significantly more fun to do.

We do this by:
1) Locking advanced engineering with BZ (organs and implants lock lifted). Depending on feedback i wont mind changing this around if you want to suggest another node as long as it's of similar or very slightly less importance.

2) Getting rid of xeno's BZ can. Some xeno players need it for making slimes sleep, with their roundstart supply removed there should be a significant demand for the BZ production in toxins to go online asap.

If you have been paying attention to our PRs, i have been working to make BZ production as seamless and quick as possible in toxins. My five map prs #66454 #66198 #66064 #66010 #65857 have been building up to this. You can make BZ relatively quickly with the new freezer chamber in place. Probably even faster than ordering it in cargo, which is a fine ballpark to use if you want to make changes to it.

If you want to know how to operate it, here is a wiki guide in place https://tgstation13.org/wiki/User:Vincentius_vin/Sandbox#BZ_Synthesis. We will move it to the main toxins page once the rest of the page is finished, pictures are added, code things are finalized etc etc.

Refer to the linked PR above for justification on the mining partner and also the reactionless explosion.

## Changelog
:cl:
balance: Made adv engi tech node require bz shells as an experiment, organs no longer need it.
balance: Adv mining no longer requires adv engi.
balance: Removed nitrium and halon shell, implant experiment lock lifted because of the former.
balance: Relocked sec 1 tech node to need pressure bombs, sec 2 no longer needs tritium bomb.
balance: Made advanced pressure bombs easier to do without funny fusion gases.
balance: Added a new mining partner that accepts smaller (even non-atmos/non-ordnance related) bombs
balance: Added more options to purchase nodes in the paper partners. Your point gain stays the same though.
balance: Removed roundstart BZ can from xenobio.
/:cl:

### CODE REVIEW

**Comment by Mothblocks** [comment]
experimental tools locked behind atmos :eyes:

**Comment by vincentiusvin** [comment]
Maybe we can move t2 tools to need something else instead of adv engi.

**Comment by moocowswag** [comment]
I think its a good opportunity to make something new for atmos techs to work towards, progression as an atmostech feels very...binary? You either manage to automate production enough to where you can do what you want before the round ends or you don`t.

Giving atmos techs a new set of tools that they could unlock by completing easy and intermediate gas experiments.

The tools being atmos/gas related.

A good example would be tools that mimic the functions of what an atmos tech would want to later automate if they had more time.


Like a tool that heats a canister/pipe or instead cooled it down and released its temperature into the room, the catch is you have to be doing it manuelly.

**Comment by vincentiusvin** [comment]
A fun idea that im sure would get accepted if PRd. Scope creep for this pr though.

**Comment by GuillaumePrata** [comment]
> balance: Removed roundstart BZ can from xenobio.

Years of Atmosian Culture die today, our younglings will never know the joy of speedrunning that BZ can and bashing a Xenobiologist's skull in for getting in the way...
Memery aside, this is a good change.

> balance: Made adv engi tech node require bz shells as an experiment, organs no longer need it.

This is an even better change and it will be fun to see Engineers complain about it instead of trying to cooperate.

**Comment by thgvr** [comment]
Once again locking unrelated job content behind things, this didn't go well for slime extract scans, it didn't go well with nitrium shells, and it won't go well with this

**Comment by vincentiusvin** [comment]
I think ill get rid of adv engi requirement for adv mining. Don't feel like we should make miners actually mining harder to do. Sounds cool maints?

**Comment by Arctrooper8** [comment]
>Removed Xenobio's BZ Can.

Please please please, don't remove Xenobio's BZ can, putting slimes to sleep is an integral process of Xenobio storage and is needed from start to finish if you want to do anything sophisticated with slimes at all.

Making it sounds great on paper, but in practice nobody is ever going to make it and the Xenobiologist is going to have to do it themselves or order it from cargo. And spending time harrassing cargo ** or having to do someone else's job just to get started on yours** isn't fun. Some people just want to do Xenobiology, and if there's no atmosian elder to brew up some BZ they'll have to beg and plead for most of the round with cargo instead of actually doing Xenobio.

Considering how important BZ has become I understand the removal of it, it's been an issue long before it was needed in research when asshole atmos techs would hack in and yoink the can.
**Personally I suggest just giving the slime sleep effect to another gas, maybe N2O? It already makes normal mobs sleep.**

TLDR: Xenobio really needs BZ to make the slimes sleep, slime sleeping is extremely important so maybe just add it to another gas?

**Comment by MetalClone** [comment]
I'm one of the few people who actually use the BZ canister in science for slimes (that being said I try to not use all of it so I can give it to toxins). Ignoring my opinions on the research papers to begin with: What if the xenobiology 'slime' storage room, just had BZ gas in it by default? I know cytologists sometimes use these rooms, so BZ gas has to go somewhere else, but I use the slime storage from around 10 minutes into the round as a xenobiologist. There doesn't even need to be a significant amount in the room by default, slimes barely need much to pacify them.

**Comment by Draggeru** [comment]
Why would you remove xenobios BZ can when xenobio uses it? Seriously this whole papers endeavor has been way too much of a headache to even bother with.

**Comment by tralezab** [comment]
conflict

**Comment by ATH1909** [comment]
> > Removed Xenobio's BZ Can.
> 
> Please please please, don't remove Xenobio's BZ can, putting slimes to sleep is an integral process of Xenobio storage and is needed from start to finish if you want to do anything sophisticated with slimes at all.
> 
> Making it sounds great on paper, but in practice nobody is ever going to make it and the Xenobiologist is going to have to do it themselves or order it from cargo. And spending time harrassing cargo ** or having to do someone else's job just to get started on yours** isn't fun. Some people just want to do Xenobiology, and if there's no atmosian elder to brew up some BZ they'll have to beg and plead for most of the round with cargo instead of actually doing Xenobio.
> 
> Considering how important BZ has become I understand the removal of it, it's been an issue long before it was needed in research when asshole atmos techs would hack in and yoink the can. **Personally I suggest just giving the slime sleep effect to another gas, maybe N2O? It already makes normal mobs sleep.**
> 
> TLDR: Xenobio really needs BZ to make the slimes sleep, slime sleeping is extremely important so maybe just add it to another gas?

do you... not use the console to move your slimes around?

**Comment by Striders13** [comment]
> do you... not use the console to move your slimes around?
Sleeping slimes don't starve to death so you don't have to throw a billion monkeys into slimes you're not currently working with

**Comment by private-tristan** [comment]
> balance: Made adv engi tech node require bz shells as an experiment, organs no longer need it.

this blocks off the experimental shears (what are they called again?) for botany, which is practically essential for any sane botanist late into the shift if you want to mess around with traits and chems :(

**Comment by vincentiusvin** [comment]
It's conflicted because maps, it's fully ready for review. I will resolve conflicts once it's discussed/approved/denied/whatever.

**Comment by vincentiusvin** [comment]
> botany shear thing

You might want to look into locking it with another node in another PR. Maybe it'll get accepted. I changed the mining one since sci and mining works together very closely and mining is pretty much critical to the station. Botany isn't as important.

I.E. I'm considering it to be out of scope but wont mind it be changed by someone else.

**Comment by LemonInTheDark** [comment]
I fundamentally agree with the idea behind this, and pushed vin in this direction

**Comment by LemonInTheDark** [comment]
> I'm one of the few people who actually use the BZ canister in science for slimes (that being said I try to not use all of it so I can give it to toxins). Ignoring my opinions on the research papers to begin with: What if the xenobiology 'slime' storage room, just had BZ gas in it by default? I know cytologists sometimes use these rooms, so BZ gas has to go somewhere else, but I use the slime storage from around 10 minutes into the round as a xenobiologist. There doesn't even need to be a significant amount in the room by default, slimes barely need much to pacify them. Sorta like the kill chambers available in every xenobiology.

Xenobio wanting BZ is kinda part of the goal here. We've made bz quite easy to create, and given scientists two initial reasons to "want it", alongside with easier production.
Especially because bz is a key component in making hypernob more easiler.

Don't really care about what's locked, but I'm ok with locking tools if it's done like this

**Comment by SmArtKar** [comment]
Imo this is a very good PR because it drops overall difficulty of gas shells while keeping endgame content for them. T4 tech is almost impossible to unlock right now but with this it would be much better since BZ is not hard to make at all.

**Comment by Redrover1760** [comment]
> > Removed Xenobio's BZ Can.
> 
> Please please please, don't remove Xenobio's BZ can, putting slimes to sleep is an integral process of Xenobio storage and is needed from start to finish if you want to do anything sophisticated with slimes at all.
> 
> Making it sounds great on paper, but in practice nobody is ever going to make it and the Xenobiologist is going to have to do it themselves or order it from cargo. And spending time harrassing cargo ** or having to do someone else's job just to get started on yours** isn't fun. Some people just want to do Xenobiology, and if there's no atmosian elder to brew up some BZ they'll have to beg and plead for most of the round with cargo instead of actually doing Xenobio.
> 
> Considering how important BZ has become I understand the removal of it, it's been an issue long before it was needed in research when asshole atmos techs would hack in and yoink the can. **Personally I suggest just giving the slime sleep effect to another gas, maybe N2O? It already makes normal mobs sleep.**
> 
> TLDR: Xenobio really needs BZ to make the slimes sleep, slime sleeping is extremely important so maybe just add it to another gas?

Actually, I played around with the new toxins setup on a test world. Making bz with it is pretty damn easy and it makes a ton, and takes 5 minutes tops if you aren't slow about it. Setting it to bz production should take 2 minutes and honestly mass producing bz for your own use in xenobio while benefiting science research is perhaps the first new sastifying thing I've done with gases in a long time.

I like it.

**Comment by Miczu555PL** [comment]
> > botany shear thing
> 
> You might want to look into locking it with another node in another PR. Maybe it'll get accepted. I changed the mining one since sci and mining works together very closely and mining is pretty much critical to the station. Botany isn't as important.
> 
> I.E. I'm considering it to be out of scope but wont mind it be changed by someone else.

Maybe lock botany stuff behind actuall botany experiments. They exist.. nobody does them.. they make surgeries cost less points?

File: code/modules/research/ordnance/scipaper_partner.dm
Comment (LemonInTheDark) [inline code/modules/research/ordnance/scipaper_partner.dm:13]: the hell is going on with this spacing

File: code/modules/research/ordnance/scipaper_partner.dm
Comment (LemonInTheDark) [inline code/modules/research/ordnance/scipaper_partner.dm:28]: so these are getting dusted?

File: code/modules/research/ordnance/scipaper_partner.dm
Comment (LemonInTheDark) [inline code/modules/research/ordnance/scipaper_partner.dm:27]: spacing

File: code/modules/research/ordnance/scipaper_partner.dm
Comment (LemonInTheDark) [inline code/modules/research/ordnance/scipaper_partner.dm:57]: see above

File: code/modules/research/ordnance/scipaper_partner.dm
Comment (LemonInTheDark) [inline code/modules/research/ordnance/scipaper_partner.dm:66]: ''

File: code/modules/research/techweb/all_nodes.dm
Comment (LemonInTheDark) [inline code/modules/research/techweb/all_nodes.dm:1263]: I don't really feel strongly about this, do as you'd like

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

File: code/modules/research/ordnance/scipaper_partner.dm
Comment (vincentiusvin) [reply code/modules/research/ordnance/scipaper_partner.dm:28]: Rolled into physics partner

File: code/modules/research/techweb/all_nodes.dm
Comment (vincentiusvin) [reply code/modules/research/techweb/all_nodes.dm:1263]: I want this 👍

File: code/modules/research/ordnance/scipaper_partner.dm
Comment (vincentiusvin) [reply code/modules/research/ordnance/scipaper_partner.dm]: Should be fixed now

**Comment by Dmeto** [comment]
> > Removed Xenobio's BZ Can.
> 
> Please please please, don't remove Xenobio's BZ can, putting slimes to sleep is an integral process of Xenobio storage and is needed from start to finish if you want to do anything sophisticated with slimes at all.
> 
> Making it sounds great on paper, but in practice nobody is ever going to make it and the Xenobiologist is going to have to do it themselves or order it from cargo. And spending time harrassing cargo ** or having to do someone else's job just to get started on yours** isn't fun. Some people just want to do Xenobiology, and if there's no atmosian elder to brew up some BZ they'll have to beg and plead for most of the round with cargo instead of actually doing Xenobio.
> 
> Considering how important BZ has become I understand the removal of it, it's been an issue long before it was needed in research when asshole atmos techs would hack in and yoink the can. **Personally I suggest just giving the slime sleep effect to another gas, maybe N2O? It already makes normal mobs sleep.**
> 
> TLDR: Xenobio really needs BZ to make the slimes sleep, slime sleeping is extremely important so maybe just add it to another gas?

Prehaps we should swap the sleep reaction of slimes to N20 and the maped canasters aswell
Anyone got any objections to this?

**Comment by MTandi** [comment]
Um, sorry, last time I played atmos, I believe you needed BZ as a catalyst for N2O formation. How should people get it now without BZ?

**Comment by Dmeto** [comment]
> Um, sorry, last time I played atmos, I believe you needed BZ as a catalyst for N2O formation. How should people get it now without BZ? And to get BZ you needed N2O.



> Um, sorry, last time I played atmos, I believe you needed BZ as a catalyst for N2O formation. How should people get it now without BZ? And to get BZ you needed N2O.

These are shells tests I believe so you wont need to form N20 in a TTV so you do get it back. You also start with 2x N20 canasters, so you aren’t in a chicken/egg scenario.

**Comment by MTandi** [comment]
> These are shells tests I believe so you wont need to form N20 in a TTV and you do get it back. You also start with 2x N20 canasters, so you aren’t in a chicken/egg scenario.

I'm thinking of sandbox missions like oldstation or bunker, where you may not have this canister. I understand that majority of players don't care, but it's simply bad to design something around a couple of canisters that SHOULD be available. Maybe we should provide some alternative way to get 5 moles of BZ to kickstart atmos from just O2, N2 and Plasma. Like some chemical reaction that makes a small amount of bz smoke or something.

**Comment by Dmeto** [comment]
> As for the slime n2o thing, i personally don't like it. I like there being a proper demand for bz to go online asap. Might fly if you convinced a maint though.
> 
> Basically take it to another pr i either don't care or agree with the suggestions.

Ya il start working on that as soon as this is merged.

**Review by Maurukas — APPROVED** [review]
Mapping is just a removal of the bz cans.

**Comment by LemonInTheDark** [comment]
Lets jive

**Comment by ThePyroTechnician** [comment]
EXPERIMENTAL TOOLS BEHIND ATMOS MY BROTHER IN CHRIST ARE YOU SURE?

**Comment by private-tristan** [comment]
experimental tools are gone
goodbye, botany shears, it was nice knowing you 😭

**Comment by vincentiusvin** [comment]
I'm pretty sure you can move the shears to techweb_node/botany and then have it not be locked by adv_engi without having the PR shot down.

**Comment by bluetshirtguy** [comment]
This pr has been quite a lot of agony man.

**Comment by Lauwma** [comment]
Me when maintainers don't play the game and think locking rcd and rpd behind atmo is sane (this is coming from an atmo main)

Yes it's easy to make, but it will never be made, and players shouldn't be forced to do things that has no end benefits. Bz is useless on its own, adding a purpose to it with science bullshit is just bad game design

**Comment by thgvr** [comment]
Gating RCDs behind an experiment combined with the new shatter charge antag objective is quite destructive to round lengths, nobody will fix a large breach with rods and floor tiles.

**Comment by vincentiusvin** [comment]
Advanced engineering is extremely bloated and a PR to move things unlocked by it (like rcd) to another node might be able to go through if the maints agree. At the very least i can say that it does not face any opposition from the ordnance side.

**Pull request merged**
